shepherd/Sensors.py

```diff
@@ -24,13 +25,14 @@ def get_working_serial_ports(excludes: set):
     """
     import glob
     maybe_ports = set(glob.glob("/dev/ttyACM*"))
+    # maybe_ports = set(glob.glob("/dev/tty.usb*"))
     maybe_ports.difference_update(excludes)
 
     working = []
     for p in maybe_ports:
         try:
             working.append(serial.Serial(p, baudrate=115200))
-        except serial.SerialException:
+        except:
```

Comment:
Why was this made to catch all exceptions? This could mask really bad problems like `NameError`. In general, caught exceptions should use the most narrow type necessary. If need be, you can define your own exceptions by subclassing `Exception`/`BaseException`.